src/Evolution/Systems/CurvedScalarWave/Worldtube/Inboxes.hpp

```diff
@@ -0,0 +1,35 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+#include <unordered_map>
+
+#include "DataStructures/Variables.hpp"
+#include "Domain/Structure/ElementId.hpp"
+#include "Parallel/InboxInserters.hpp"
+#include "Time/TimeStepId.hpp"
+
+namespace CurvedScalarWave::Worldtube::Tags {
+
+/*!
+ * \brief Inbox of the worldtube singleton chare which receives quantities
+ * projected onto spherical harmonics.
+ *
+ * \details Each element abutting the worldtube projects both `Psi` and the time
+ * derivative of `Psi` onto spherical harmonics by integrating it over the
```

Comment:
change to \f$ \Psi\f$

Reply:
FYI you can also write just `$\Psi$` (we have a Doxygen filter that understands standard LaTeX)